Assets/MixedRealityToolkit.Tests/EditModeTests/Experimental/MigrationToolTest.cs

```diff
@@ -0,0 +1,115 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Experimental.UI;
+using Microsoft.MixedReality.Toolkit.Experimental.Utilities;
+using Microsoft.MixedReality.Toolkit.UI;
+using NUnit.Framework;
+using System;
+using UnityEditor;
+using UnityEditor.SceneManagement;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+namespace Microsoft.MixedReality.Toolkit.Tests
+{
+    public class MigrationToolTest
+    {
+        private readonly MigrationTool migrationTool = new MigrationTool();
+
+        /// <summary>
+        /// Checks if MigrationTool can process migration on a GameObject containing a deprecated ManipulationHandler component.
+        /// </summary>
+        [Test]
+        public void GameObjectCanBeMigrated()
+        {
+            Type oldType = typeof(ManipulationHandler);
+            Type newType = typeof(ObjectManipulator);
+            Type migrationHandlerType = typeof(ObjectManipulatorMigrationHandler);
+
+            GameObject gameObject = SetUpGameObjectWithComponentOfType(oldType);
+
+            migrationTool.TryAddObjectForMigration(gameObject);
+            migrationTool.MigrateSelection(migrationHandlerType, false);
```

Comment:
It would be great if there was a way we could add some tests that migrate a prefab, or a scene too. Is this something that's possible to test here?

Reply:
I believe those test are already there no? There is a test where a prefab asset is created from that game object. and then the actual prefab asset is migrated. The same for scene asset.

Reply:
omg, sorry. You're totally right, haha. Somehow totally missed the other tests in my first review. Apologies for that.